core/src/main/scala/pureconfig/ConfigConvert.scala

```diff
@@ -254,7 +273,7 @@ object ConfigConvert extends LowPriorityConfigConvertImplicits {
   }
 
   // used for products
-  implicit def deriveInstanceWithLabelledGeneric[F, Repr <: HList, DefaultRepr <: HList](
+  implicit def deriveProductInstance[F, Repr <: HList, DefaultRepr <: HList](
```

Comment:
Yeah, I understand your preference and I also have that personal policy :) In this case I think you really had to change something as otherwise you'd end up with two methods, `deriveInstanceWithLabelledGeneric` and `deriveInstanceWithGeneric`, both using now `LabelledGeneric`. But if you can think of other, less "disruptive" names I can change it without a problem.